test/setup-beam.test.js

```diff
@@ -542,7 +605,7 @@ async function testParseVersionFile() {
 erlang   ref:v${erlang}# comment, no space, and ref:v
 elixir ref:${elixir}  # comment, with space and ref:
  not-gleam 0.23 # not picked up
-gleam ${gleam} 
```

Comment:
This should be reversed. It's part of the test to make sure whitespace is not important in the parsing of the file.

Reply:
Yeah, I'm not sure where or how to change this in Zed, and I assume it's a bug since this is happening in a string 

cc @zed-industries